diff --git a/node_modules/cdk8s-cli/lib/cli/cmds/synth.js b/node_modules/cdk8s-cli/lib/cli/cmds/synth.js
index fba8be7..375af92 100644
--- a/node_modules/cdk8s-cli/lib/cli/cmds/synth.js
+++ b/node_modules/cdk8s-cli/lib/cli/cmds/synth.js
@@ -103,9 +103,7 @@ class Command {
         if (stdout) {
             await (0, util_1.mkdtemp)(async (tempDir) => {
                 const app = await (0, util_1.synthApp)(command, tempDir, stdout, recordConstructMetadata);
-                for (const f of app.manifests) {
-                    fs.createReadStream(f).pipe(process.stdout);
-                }
+                process.stdout.write(app.manifests.map(m => fs.readFileSync(m, 'utf8')).join('---\n'))
                 if (validations) {
                     const pluginManager = new _manager_1.PluginManager(pluginsDir);
                     await (0, util_1.validateApp)(app, stdout, validations, pluginManager, reportFile);
